torch/nn/modules/upsampling.py

```diff
@@ -184,46 +156,36 @@ def forward(self, input):
 class UpsamplingBilinear2d(Upsample):
     r"""Applies a 2D bilinear upsampling to an input signal composed of several input
     channels.
-
     To specify the scale, it takes either the :attr:`size` or the :attr:`scale_factor`
     as it's constructor argument.
-
     When `size` is given, it is the output size of the image `(h, w)`.
-
     Args:
         size (tuple, optional): a tuple of ints `(H_out, W_out)` output sizes
         scale_factor (int, optional): the multiplier for the image height or width
-
     .. warning::
         This class is deprecated in favor of :func:`~nn.functional.interpolate`. It is
         equivalent to ``nn.functional.interpolate(..., mode='bilinear', align_corners=True)``.
-
     Shape:
         - Input: :math:`(N, C, H_{in}, W_{in})`
         - Output: :math:`(N, C, H_{out}, W_{out})` where
-
     .. math::
         H_{out} = \left\lfloor H_{in} \times \text{scale\_factor} \right\rfloor
-
     .. math::
         W_{out} = \left\lfloor W_{in} \times \text{scale\_factor} \right\rfloor
-
     Examples::
-
         >>> input = torch.arange(1, 5).view(1, 1, 2, 2)
         >>> input
         tensor([[[[ 1.,  2.],
                   [ 3.,  4.]]]])
-
         >>> m = nn.UpsamplingBilinear2d(scale_factor=2)
         >>> m(input)
         tensor([[[[ 1.0000,  1.3333,  1.6667,  2.0000],
                   [ 1.6667,  2.0000,  2.3333,  2.6667],
                   [ 2.3333,  2.6667,  3.0000,  3.3333],
                   [ 3.0000,  3.3333,  3.6667,  4.0000]]]])
     """
-    def __init__(self, size=None, scale_factor=None):
-        super(UpsamplingBilinear2d, self).__init__(size, scale_factor, mode='bilinear', align_corners=True)
```

Comment:
This is the change I wanted to make. Adding align_corners parameter for UpsamplingBilinear2d and set the default value to False as specified in the documentation: https://pytorch.org/docs/stable/_modules/torch/nn/modules/upsampling.html. It would cause problems especially when you are trying to use ONNX to convert UpsamplingBilinear2d layers.

Reply:
This is going to change the default behavior and will probably break exisitng users of UpsamplingBilinear2D. Is there a reason why you want the default to be `align_corners=False`?

Reply:
In the comments section in Upsample class, it says
```python
'''
align_corners (bool, optional): if True, the corner pixels of the input
            and output tensors are aligned, and thus preserving the values at
            those pixels. This only has effect when :attr:`mode` is `linear`,
            `bilinear`, or `trilinear`. Default: False
'''
```
So I thought the default value for UpsamplingBilinear2D's align_corners should be False as well. Would it be better that switch it back to True?

Reply:
I'm looking at the history. Based on #5927 it looks like we might have forgotten to update the docs. cc @SsnL what should be the default for align_corners?

Reply:
Also, it might be worth double-checking that, whether ONNX supports align_corners==False or align_corners==True for converting UpsamplingBilinear2d? Currently, it supports align_corners==False only.
```python
def upsample_bilinear2d(g, input, output_size, align_corners):
    if align_corners:
        return _unimplemented("upsample_bilinear2d", "align_corners == True")
    height_scale = float(output_size[-2]) / input.type().sizes()[-2]
    width_scale = float(output_size[-1]) / input.type().sizes()[-1]
    return g.op("Upsample", input,
                scales_f=[1., 1., height_scale, width_scale],
                mode_s="bilinear")
```

Reply:
Nah the docs are correct. Please keep `UpsamplingBilinear2D`'s behavior as-is. That class is deprecated and we shouldn't be changing it's behavior.

I'm also hesitant to adding a new option considering that it is deprecated. Not being onnx-export-able shouldn't be a problem because people should use `Upsamling` instead.